bokehjs/test/integration/regressions.ts

```diff
@@ -1493,6 +1493,31 @@ describe("Bug", () => {
     })
   })
 
+  describe("in issue #11462", () => {
+    it("doesn't update ColorBar after mapper/axis/title property updates", async () => {
+      const random = new Random(1)
+      const p = fig([200, 200])
+
+      const color_mapper = new LinearColorMapper({palette: turbo(50), low: 0, high: 1})
+      const color_bar = new ColorBar({color_mapper, title: "original title", label_standoff: 12})
+      p.add_layout(color_bar, "right")
+
+      const dw = 10
+      const dh = 10
+      const img = ndarray(random.floats(dw*dh), {dtype: "float64", shape: [dw, dw]})
+      p.image({image: [img], x: 0, y: 0, dw, dh, color_mapper})
+
+      const {view} = await display(p, [350, 350])
+      await view.ready
```

Comment:
@g-parki, awaiting `ready` promise is not needed just after `await display`, as those are synonymous. I removed this in PR #11780.